doc/api/training/smd_model_parallel_general.rst

```diff
@@ -214,6 +243,34 @@ PyTorch-specific Parameters
     - False
     - Skips the initial tracing step. This can be useful in very large models
       where even model tracing at the CPU is not possible due to memory constraints.
+  * - ``sharded_data_parallel_degree`` (**smdistributed-modelparallel**>=v1.11)
+    - int
+    - 1
```

Comment:
please confirm if this is the default value.

Reply:
Is this resolved?

Reply:
Yes, the default value is 1